app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe_food, only: %i[show edit update destroy]
+
+  def index
+    @recipe_foods = RecipeFoods.all
+  end
+
+  def new
+    @recipe_food = RecipeFood.new
+  end
+
+  def create
+    @recipe = Recipe.find(params[:recipe_id])
+    @recipe_food = @recipe.recipe_foods.new(recipe_food_params)
+    @recipe_food.save
+  end
```

Comment:
When I add a new record it doesn't redirect to another page or display a success massage, it's better to add a way to know the user his work progress.